.github/workflows/e2e-tests.yaml

```diff
@@ -238,5 +238,6 @@ jobs:
               --repository-name ${REPOSITORY_NAME} \
               --image-tag ${TAG_PASSED}-linux-${ARCH}-amazon \
               --image-manifest "$MANIFEST" \
-              --region ${AWS_REGION}
-          done
\ No newline at end of file
+              --region ${AWS_REGION} 2>&1 | grep -q "ImageAlreadyExistsException" || \
+              { err=$?; [ $err -eq 1 ] || exit $err; }
```

Comment:
What does this bit do?

Reply:
Seems a bit cryptic, but I think it just tries to ignore `ImageAlreadyExistsException` errors otherwise just propagates the error from the original command. I'm okay with the change to fix the CI but would be nice if we can add a comment explaining what this bit does and why.